src/components/Filter/Filter.tsx

```diff
@@ -1,45 +1,52 @@
+import { useOnClickOutside } from "usehooks-ts";
+import { useRef, useState } from "react";
+import { Content, Dropdown, Header, Item } from "./style";
+import dropdownIcon from "../../assets/icons/dropdown.svg";
 
-import { useState } from 'react';
-import { Content, Dropdown, Header, Item } from './style';
-import dropdownIcon from '../../assets/icons/dropdown.svg';
-
-export interface FilterProps{
-    options: string[],
-    name: string,
-    onChangeValue: (value: string) => void,
+export interface FilterProps {
+  options: string[];
```

Comment:
It may be better to get an object's array like : 
{
 value : number | string;
name : string;
}

rather the string[]